### ISSUE TITLE
OGB Hyperparameters

### ISSUE THREAD

**Issue by Timob12** [allenhaozhu/SSGC#12]
Hello,
are there any specific hyperparameters one should use when trying to reproduce the OGB node classification results? I extended the approach from the cora, citeseer, and pubmed dataset to the OGB-arxiv dataset but only achieve ~50% accuracy and not 70% as stated in the paper. This is especially weird because the cora, citeseer, and pubmed results are all agreeing with the ones in the paper. Also, your link to the preprocessed files is not working ("https://drive.google.com/drive/u/0/my-drive").

**Comment by allenhaozhu** [comment]
Sorry, can you share with me how do you train ogb-arxiv? Usually, I will use ssgc to get embedding and then train a mlp as a classifier. That's why I provide two different files

**Comment by Timob12** [comment]
Sorry, I made a mistake when multiplying the matrices. It is working now and I get similar results. However, the link to the preprocessed files in you document classification directory is still not accessable.

**Comment by allenhaozhu** [comment]
> Sorry, I made a mistake when multiplying the matrices. It is working now and I get similar results. However, the link to the preprocessed files in you document classification directory is still not accessable.

I guess you need to generate intermediate files by yourself because these files are too big. I follow the guideline from TextGCN.
